dist-scripts/lib/crossmintStatus.js

```diff
@@ -0,0 +1,26 @@
+export const isAuthLoading = (status) => {
+    if (!status)
+        return false;
```

Comment:
we hvae this same logic in multiple files, can we move this to a util so we can re-use